### PR TITLE
Implemented "Show a message when no documents are attached yet." #641

### PR DESCRIPTION
**Pull request by arispapapro**

### CODE REVIEW

File: csc-courier-ssapp/code/scripts/controllers/EditShipmentController.js [hunk 1]
```diff
@@ -58,13 +58,17 @@ export default class EditShipmentController extends WebcController {
           });
         });
       }
+
+      this.model.form.areFilesEmpty = (this.files.length <= 0);
```
Comment (Mastaleru) [inline csc-courier-ssapp/code/scripts/controllers/EditShipmentController.js:62]: inadequate comparation. arrays length cannot be a negative number

File: csc-courier-ssapp/code/scripts/controllers/EditShipmentController.js [hunk 2]
```diff
@@ -263,6 +267,8 @@ export default class EditShipmentController extends WebcController {
 	this.model.shipment = this.originalShipment;
     this.model.form.documents = [];
     this.model.wizard = this.getWizardForm();
+    this.model.form.areFilesEmpty = true;
```
Comment (Mastaleru) [inline csc-courier-ssapp/code/scripts/controllers/EditShipmentController.js:270]: areFilesEmpty-> hasAttachments

https://www.samanthaming.com/tidbits/34-better-boolean-variable-names/

File: csc-courier-ssapp/code/pages/edit-shipment.html [hunk 3]
```diff
@@ -64,8 +64,9 @@
               <psk-files-chooser event-name="add-file" data-view-model="@form.attachment"></psk-files-chooser>
               <p>Maximum upload file size: 64MB</p>
             </div>
-            <div class="col-12">
-              <table class="table table-borderless attach-documents-table">
+            <div data-if="@form.areFilesEmpty"  class="col-12">
+              <p class='text-center' slot="true">There are no any Files uploaded yet</p>
```
Comment (Mastaleru) [inline csc-courier-ssapp/code/pages/edit-shipment.html:68]: use bootstrap light alert as you proposed

**Review by Mastaleru — CHANGES_REQUESTED** [review]

**Comment by arispapapro** [comment]
I made the changes rafael you can merge it.

**Review by Mastaleru — APPROVED** [review]

**Pull request merged**